internal/budgetadjustments/events/examples/delete_example.sh

```diff
@@ -1,11 +1,26 @@
-# Delete Adjustment Evens using a file
-sloctl budgetadjustments events delete --adjustment-name=sample-adjustment-name -f /path/to/file/with/events
+# Delete Adjustment Events using a file
+sloctl budgetadjustments events delete --adjustment-name=sample-adjustment-name -f /path/to/file/with/events/yaml
```

Comment:
you can delete it since you already have that at line 13 :)

Reply:
rearranged